lib/ui/window.dart

```diff
@@ -210,7 +210,13 @@ class Window {
   /// [physicalSize], or [padding] values change, for example when the device is
   /// rotated or when the application is resized (e.g. when showing applications
   /// side-by-side on Android).
-  VoidCallback onMetricsChanged;
+  VoidCallback get onMetricsChanged => _onMetricsChanged;
+
+  set onMetricsChanged(VoidCallback callback) {
+    _onMetricsChanged = _zonedVoidCallback(callback);
+  }
+
+  VoidCallback _onMetricsChanged;
```

Comment:
Flutter style is to have the getter, the field, then the setter, in that order, with no whitespace between them.